sdk/eventhub/azure-eventhubs/azure/eventhub/client_abstract.py

```diff
@@ -82,54 +82,54 @@ def _build_uri(address, entity):
-        :type address: str
-        :param username: The name of the shared access policy. This must be supplied
-         if not encoded into the address.
-        :type username: str
-        :param password: The shared access key. This must be supplied if not encoded
-         into the address.
-        :type password: str
-        :param debug: Whether to output network trace logs to the logger. Default
+        Constructs a new EventHubClient.
+
+        :param host: The hostname URI string of the the Event Hub.
+        :type host: str
+        :param event_hub_path: The path/name of the Event Hub
+        :type event_hub_path: str
+        :param network_tracing: Whether to output network trace logs to the logger. Default
          is `False`.
-        :type debug: bool
+        :type network_tracing: bool
+        :param credential: The credential object used for authentication which implements particular interface
+         of getting tokens. It accepts ~azure.eventhub.EventHubSharedKeyCredential,
+         ~azure.eventhub.EventHubSASTokenCredential, credential objects generated by the azure-identity library and
+         objects that implement get token interface.
         :param http_proxy: HTTP proxy settings. This must be a dictionary with the following
          keys: 'proxy_hostname' (str value) and 'proxy_port' (int value).
          Additionally the following keys may also be present: 'username', 'password'.
         :type http_proxy: dict[str, Any]
         :param auth_timeout: The time in seconds to wait for a token to be authorized by the service.
          The default value is 60 seconds. If set to 0, no timeout will be enforced from the client.
         :type auth_timeout: int
-        :param sas_token: A SAS token or function that returns a SAS token. If a function is supplied,
-         it will be used to retrieve subsequent tokens in the case of token expiry. The function should
-         take no arguments.
-        :type sas_token: str or callable
+        :param user_agent: The user agent that needs to be appended to the built in user agent string.
+        :type user_agent: str
+        :param max_retries: The max number of attempts to redo the failed operation when an error happened. Default
+         value is 3.
+        :type max_retries: int
+        :param transport_type: The transport protocol type - default is ~uamqp.TransportType.Amqp.
+         ~uamqp.TransportType.AmqpOverWebsocket is applied when http_proxy is set or the
+         transport type is explicitly requested.
+        :type transport_type: ~azure.eventhub.TransportType
+        :param prefetch: The message prefetch count of the receiver. Default is 300.
+        :type prefetch: int
+        :param send_timeout: The timeout in seconds for an individual event to be sent from the time that it is
+         queued. Default value is 60 seconds. If set to 0, there will be no timeout.
+        :type send_timeout: int
```

Comment:
Is this really an `int`? I would expect to be able to pass in floats as well...

Reply:
float works. the docstring is wrong. Will correct it.